### DISCUSSION TITLE
Allowed Amount - Claim Modifier - Schema location for 0.10.1 seems incorrect

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#361] [Q&A] [answered]
Shouldn't the billing_code_modifier be part of the out of network payments object instead of allowed amount object?  It was added to the negotiated rate object in the In-Network schema, so wouldn't the alignment for the Allowed Amount be in the payments object?

Here's an example---where if the allowed amounts vary based on modifier, then it can be noted for the given item/service.  

Seems if it remains in the allowed amount object, then you would have to repeat some of the same data for a given procedure that may have base allowed amounts different than those with modifiers for the same procedure.

Can someone clarify why the claim modifier is defined in allowed amount object rather than the payments object for the Allowed Amount File?  @shaselton-usds  

Here's what I thought would be more in alignment with how the data schema aligned with pricing in the In-Network schema---where the modifier would be part of the payment data.   Thoughts??
{
	"reporting_entity_name": "cms",
	"reporting_entity_type": "cms",
	"plan_name": "medicare",
	"plan_id_type": "hios",
	"plan_id": "11111111111",
	"plan_market_type": "individual",
	"last_updated_on": "2020-08-27",
	"version": "1.0.0",
	"out_of_network": [{
		"name": "Established Patient Office or Other Outpatient Services",
		"billing_code_type": "CPT",
		"billing_code_type_version": "2020",
		"billing_code": "99214",
		"description": "office or other outpatient visits for the evaluation and management of an established patient, which requires at least two of these three key components: a detailed history, a detailed examination and medical decision making of moderate complexity",
		"allowed_amounts": [{
			"tin": {
				"type": "ein",
				"value": "1234567890"
			},
			"service_code": ["01", "02", "03"],
			"billing_class": "professional",
			"payments": [{
				"allowed_amount": 25.00,
				"providers": [{
					"billed_charge": 50.00,
					"npi": [1234567891, 1234567892, 1234567893]
				}, {
					"billed_charge": 60.00,
					"npi": [1111111111]
				}, {
					"billed_charge": 70.00,
					"npi": [2222222222, 3333333333, 4444444444, 5555555555],
					"allowed_amount": 30.00,
					**"billing_code_modifier": ["25"],**
					"providers": [{
						"billed_charge": 25.00,
						"npi": [3434343434]
					}]
				}]
			}]
		}]
	}]
}

**Comment by shaselton-usds** [comment, marked as answer]
@tdfow This definitely makes sense. I'm not sure how the PR ended up where it did because this was the original intend. Thank you for the watchful eye!

**Reply by shaselton-usds** [reply]
Thanks again for the catch. PR here: https://github.com/CMSgov/price-transparency-guide/pull/373

**Reply by tdfow** [reply]
@shaselton-usds when will this be updated in the master branch?

**Reply by tdfow** [reply]
CMS has indicated we should use the master branch for the schemas, yet the develop branch has had the bug corrections for nearly a month without the master branch being updated.   Seems a bug correction would be applied to the master branch sooner than later to avoid further delays and confusion for those using the master schema version.     Can we get a timeline for the bug correction to the master?
